### PR TITLE
Removed salmon due to allergies, fixes #1

### LINKED ISSUE #1: Salmon allergy (sides/poke.md)
Everyone has salmon allergy, we need to remove salmon from the ingredients list

### PR DESCRIPTION
**Pull request by josefmtd**
Tuna is better and fattier

### CODE REVIEW

File: sides/poke.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 ## Ingredients
 
 - avocado
-- salmon or ahi-tuna
+- ahi-tuna
```
Comment (eglerean) [inline sides/poke.md:7]: salmon is the name of our company, try to bring it back

**Review by eglerean — COMMENTED** [review]
check my comments

File: sides/poke.md (hunk 1 above)
Comment (josefmtd) [reply sides/poke.md]: I've now added lab-grown salmon as a substitute

**Review by MisbahSaleemm — COMMENTED** [review]
lab-grown salmon could be expensive

**Pull request merged**